src/components/pages/dashboard.tsx

```diff
@@ -37,16 +44,16 @@ const getValue = getValueFromTheme('dashboardPage');
 
 const itemsPerPage = 14;
 
-const GetItemsByCreatorMap = {
+const UseGetItemsByCreatorMap = {
   events: useGetEventsByCreator,
   places: useGetPlacesByCreator,
   organizers: useGetOrganizersByCreator,
 } as const;
 
-const DeleteEventByIdMap = {
+const UseDeleteEventByIdMap = {
```

Comment:
Why is this specifically called UseDelete**Event**ByIdMap?
Because you can also use this map to delete organizers, events and places.
Isn't `UseDeleteByIdMap` sufficient?

Reply:
True this is a typo left over 😄